src/app/app-modules/core/components/app-header/app-header.component.ts

```
@@ -98,9 +104,36 @@ export class AppHeaderComponent implements OnInit, OnChanges {
     });
   }
 
+  refreshLogin() {
+    this.auth.getUserDetails().subscribe((res: any) => {
+      if (res.statusCode === '200') {
+        if (res.data?.previlegeObj[0]) {
+          this.cookieService.set('Jwttoken', res.data.Jwttoken);
+          delete res.data.Jwttoken;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)